dcos_launch/terraform.py

```diff
@@ -0,0 +1,100 @@
+from dcos_launch import util
+import yaml
+import os
+import json
+import shutil
+import uuid
+
+# TODO pinning terraform-dcos is not enough because its tf_dcos_core dependency is not pinned. Create a branch on
+# terraform-dcos that pins tf_dcos_core and replace TERRAFORM_DCOS_VERSION here by that branch name. Regularly update
+# that branch with master.
+TERRAFORM_DCOS_VERSION = 'master'
+TERRAFORM_DCOS_EE_VERSION = 'service_credentials_gcp2'
```

Comment:
I think these should be configurable fields in dcos_launch/config.py

Reply:
+1

Reply:
Done